app/controllers/v0/sessions_controller.rb

```diff
@@ -146,19 +138,6 @@ def build_logout_errors(logout_response, logout_request)
       errors
     end
 
-    def benchmark_tags(*tags)
```

Comment:
benchmark_tags was added [here](https://github.com/department-of-veterans-affairs/vets-api/pull/1417/files) and last used  [here](https://github.com/department-of-veterans-affairs/vets-api/commit/9b076c178c4069f00f19adcee2477223ce4baef4#diff-9ceac19063f82b7a0b6cd02f819e0b62)

Reply:
so why is it being removed exactly? It's not immediately clear to me looking at the code.

Reply:
i see, it's not getting used, but we should probably to look to add this functionality back as its really useful information to build SLI around.

Reply:
I opened up this ticket to revisit later: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15585